opentech/apply/users/forms.py

```diff
@@ -18,3 +22,26 @@ class CustomUserEditForm(CustomUserAdminFormBase, UserEditForm):
 
 class CustomUserCreationForm(CustomUserAdminFormBase, UserCreationForm):
     pass
+
+
+class ProfileForm(forms.ModelForm):
+    class Meta:
+        model = User
+        fields = ['full_name', 'email', 'slack']
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        if not self.instance.is_apply_staff:
+            del self.fields['slack']
+
+    def clean_slack(self):
+        slack = self.cleaned_data['slack']
+        if slack:
+            slack = slack.strip()
+            if ' ' in slack:
+                raise forms.ValidationError('Slack IDs must not include spaces')
```

Comment:
consistency: the field is labeled as "Slack name" on the form, but the error references IDs